index.js

```diff
@@ -338,6 +336,11 @@ const parseModel = (name,model) => {
   let schema = typeof model.schema === 'object' && !Array.isArray(model.schema) ?
     model.schema : error(`Please provide a valid 'schema'`)
 
+  let delimiter = typeof model.delimiter === 'string'
+    && model.delimiter.trim().length > 0 ? model.delimiter.trim()
+    : model.delimiter ? error(`'delimiter' must be string value`)
+    : '#'
```

Comment:
This is a little rough on the eyes.... perhaps this would read better?
```
let delimiter = '#'
if (typeof model.delimiter === 'string' && model.delimiter.trim().length > 0) {
  delimiter = model.delimiter.trim()
} else if (model.delimiter) error(`'delimiter' must be string value with length >= 1`)
```

Reply:
Good call. I've made that change.